deps/build.jl

```diff
@@ -4,6 +4,8 @@ using MPI
 import Pkg.TOML
 import P4est_jll
 
+const pre_generated_bindings_filename = joinpath(@__DIR__, "libp4est-2.3.1.jl")
```

Comment:
This hard-codes the exact version number (equality vs. tilde specifier above)

Reply:
Yes, I thought this would make it easier in the future to understand from which exact version these bindings have been generated, in case a subsequent release of p4est breaks stuff.